AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -184,15 +184,11 @@ internal final class ComponentManager {
     }
     
     private func createBoletoComponent(_ paymentMethod: BoletoPaymentMethod) -> BoletoComponent {
-        BoletoComponent(
-            configuration: BoletoComponent.Configuration(
-                boletoPaymentMethod: paymentMethod,
-                payment: configuration.payment,
-                shopperInfo: configuration.shopper,
-                showEmailAddress: true
-            ),
-            apiContext: apiContext
-        )
+        let boletoConfiguration = BoletoComponent.Configuration(boletoPaymentMethod: paymentMethod,
+                                                                payment: configuration.payment,
+                                                                shopperInformation: configuration.shopper,
+                                                                showEmailAddress: true)
+        return BoletoComponent(configuration: boletoConfiguration, apiContext: apiContext)
```

Comment:
can we create the `configuration` when passing it into the `BoletoComponent.init` and make this a one-statement function?

Reply:
You mean like this:

```swift
    private func createBoletoComponent(_ paymentMethod: BoletoPaymentMethod) -> BoletoComponent {
        BoletoComponent(configuration: .init(boletoPaymentMethod: paymentMethod,
                                             payment: configuration.payment,
                                             shopperInformation: configuration.shopper,
                                             showEmailAddress: true),
                        apiContext: apiContext)
    }
```

Reply:
yes, exactly like this 👍